sql/catalyst/src/main/antlr3/org/apache/spark/sql/catalyst/parser/SparkSqlLexer.g

```diff
@@ -465,7 +467,7 @@ Identifier
 fragment    
 QuotedIdentifier 
     :
-    '`'  ( '``' | ~('`') )* '`' { setText(getText().substring(1, getText().length() -1 ).replaceAll("``", "`")); }
+    '`'  ( '``' | ~('`') )* '`' { setText(getText().replaceAll("``", "`")); }
```

Comment:
So we want backticks at the beginning and the end of the identifier? I thought the first and the last backtick are a means of identifying a quoted identifier, and not a part of the name. Do these backticks remain a part of the name throughout the code?